ansible-fmw/wls_variables.yml

```diff
@@ -15,7 +15,7 @@ JDK_DIR: '{{ JAVA_DIR }}/jdk1.8.0_331'
 JAVA_HOME: '{{ JAVA_DIR }}/java_latest'
 
 ## Oracle ENV Variables
-ORACLE_BASE: "/home/dojowls/wls_lite"
+ORACLE_BASE: "/home/dojowls/day-3"
```

Comment:
IMHO we should avoid using custom names like `day-3`  
the should be generic and reusable, e.g. `wls-server` 
remember that idempotency is important aspect of IaC

Reply:
This has been changed to `ORACLE_BASE: "/home/{{ WLS_USER }}/fmw" `